Fixes/Ports from upstream/side(?)stream

MarkSuckerberg opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Gets everything recently added to WS, as well as a few fix ports from Singulostation.

singulostation13/singulostation13#6
singulostation13/singulostation13#30
singulostation13/singulostation13#39

## Changelog
:cl:
fix: Royal moth wings
fix: Partially fixed latejoin cryopod bug
fix: wallmounts now fit on both sides of a 1-wide corridor
spellcheck: A few typo fixes
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Redory committed:
Fixed haswallmount and structure/sign dir (#6)

KubeRoot committed:
Path of least resistance - teleport to location first, try to put into sleeper second in case that fails, and remove destroyed sleepers from latejoin locations (#39)

Terra1 committed:
Merge pull request #30 from Urumasi/typo

Merge requested by maintainer.

Whitesands-Bot committed:
Automatic changelog compile [ci skip]

Whitesands-Bot committed:
Automatic changelog compile [ci skip]

Whitesands-Bot committed:
Automatic changelog compile [ci skip]

Whitesands-Bot committed:
Automatic changelog compile [ci skip]

SynnGraffkin committed:
Update DeltaStation2.dmm (#810)

The failed check is caused by the integration tests loading WS instead of LL.  For some reason I am unable to rerun this check, so I am overriding it.

Whitesands-Bot committed:
Automatic changelog generation for PR #810 [ci skip]

Whitesands-Bot committed:
Automatic changelog compile [ci skip]

Whitesands-Bot committed:
Automatic changelog compile [ci skip]

1glitchycent commented:
![image](https://user-images.githubusercontent.com/57820766/117001604-8fb29880-acb0-11eb-828a-3854b0eb2885.png)

MarkSuckerberg commented on whitesands/code/datums/ruins/jungle.dm:
```diff
@@ -14,3 +14,9 @@
 	description = "Ook. Ooh Ooh Aah. AAH. OOK OOK OOK. OOK OOK AAH AAAH AAAAAAAAAAAAAAAH!"
 	suffix = "jungle_surface_monkies.dmm"
+
+/datum/map_template/ruin/jungle/solgov_crash
```
```suggestion
/datum/map_template/ruin/jungle/ambush
```

MarkSuckerberg commented:
Also, you'll need to mapmerge the map. Ask on discord if you don't know how

Whitesands-Bot committed:
Automatic changelog compile [ci skip]

Vorak2 committed:
Air alarms and reported problems (#809)

Whitesands-Bot committed:
Automatic changelog generation for PR #809 [ci skip]

dependabot[bot] committed:
Bump pyyaml from 5.3.1 to 5.4 in /tools (#823)

Bumps [pyyaml](https://github.com/yaml/pyyaml) from 5.3.1 to 5.4.
- [Release notes](https://github.com/yaml/pyyaml/releases)
- [Changelog](https://github.com/yaml/pyyaml/blob/master/CHANGES)
- [Commits](https://github.com/yaml/pyyaml/compare/5.3.1...5.4)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

untrius committed:
fixed royal moth wings (#840)

Co-authored-by: Untrius <untrius@protonmail.com>

Whitesands-Bot committed:
Automatic changelog generation for PR #840 [ci skip]

LastCrusader105 committed:
Plasmeme command helmets buff (#798)

Co-authored-by: Untrius <untrius@protonmail.com>

1glitchycent commented:
Taking break from SS13, will post map file in discord if anyone wants to PR it. If not, I'll come back to it in June.

untrius committed:
Fixes Clown Flower Superlube (#839)

* clown flower transfers superlube

* clean up

Co-authored-by: Untrius <untrius@protonmail.com>

Shadowtail117 committed:
Bone medipen now works (#846)

DerFlammenwerfer committed:
Adds Enginseer Robe (#847)

* Adds Enginseer Robe

It's a variant of the normal techpriest robe designed to look better than the original, without removing the original for those who prefer it.

* Should fix certain errors I missed in the initial PR

* Update code/modules/vending/wardrobes.dm

Margularization be damned.

Co-authored-by: Shadow <shadowtail117@gmail.com>

Co-authored-by: Shadow <shadowtail117@gmail.com>

dependabot[bot] committed:
Bump pillow from 7.2.0 to 8.1.1 in /tools (#812)

Bumps [pillow](https://github.com/python-pillow/Pillow) from 7.2.0 to 8.1.1.
- [Release notes](https://github.com/python-pillow/Pillow/releases)
- [Changelog](https://github.com/python-pillow/Pillow/blob/master/CHANGES.rst)
- [Commits](https://github.com/python-pillow/Pillow/compare/7.2.0...8.1.1)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Jack7D1 committed:
Adds the unfollow verb to the mentor tab. (#816)

* Adds the unfollow verb to the mentor tab.

* nice one

MarkSuckerberg committed:
Merge remote-tracking branch 'ws/master' into fixes
